tmt/steps/report/junit.py

```diff
@@ -101,7 +299,35 @@ class ReportJUnitData(tmt.steps.report.ReportStepData):
         option='--file',
         metavar='PATH',
         help='Path to the file to store JUnit to.',
-        normalize=lambda key_address, raw_value, logger: Path(raw_value) if raw_value else None)
+        normalize=tmt.utils.normalize_path)
+
+    flavor: str = field(
+        default=DEFAULT_FLAVOR_NAME,
+        option='--flavor',
+        choices=[DEFAULT_FLAVOR_NAME, CUSTOM_FLAVOR_NAME],
+        help=f"""
+            Name of a JUnit flavor to generate. By default, the '{DEFAULT_FLAVOR_NAME}' flavor
+            is used.
```

Comment:
This looks weird in the generated documentation:
```
flavor: default|custom

    Name of a JUnit flavor to generate. By default, the ‘default’ flavor is used.

    Default: default
```
```suggestion
            Name of a JUnit flavor to generate.
```

Reply:
Thanks, modified.